cmd/harmony/main.go

```diff
@@ -182,35 +183,24 @@ func initSetup() {
 }
 
 func setUpConsensusKeyAndReturnIndex(nodeConfig *nodeconfig.ConfigType) (int, *genesis.DeployAccount) {
-	// If FN node running, they should either specify blsPrivateKey or the file with passphrase
-	if *blsKeyFile != "" && *blsPass != "" {
-		passPhrase, err := utils.GetPassphraseFromSource(*blsPass)
-		if err != nil {
-			fmt.Printf("error when reading passphrase file: %v\n", err)
-			os.Exit(100)
-		}
-		consensusPriKey, err := blsgen.LoadBlsKeyWithPassPhrase(*blsKeyFile, passPhrase)
-		if err != nil {
-			fmt.Printf("error when loading bls key, err :%v\n", err)
-			os.Exit(100)
-		}
-		index, acc := genesis.IsBlsPublicKeyIndex(consensusPriKey.GetPublicKey().SerializeToHexStr())
-		if index < 0 {
-			fmt.Println("Can not found your bls key.")
-			os.Exit(100)
-		}
+	consensusPriKey, err := blsgen.LoadBlsKeyWithPassPhrase(*blsKeyFile, blsPassphrase)
+	if err != nil {
+		fmt.Printf("error when loading bls key, err :%v\n", err)
+		os.Exit(100)
+	}
+	index, acc := genesis.IsBlsPublicKeyIndex(consensusPriKey.GetPublicKey().SerializeToHexStr())
+	if index < 0 {
+		fmt.Println("Can not found your bls key.")
```

Comment:
Can not find*

Reply:
LOL that was just moved from `setupConsensusAndNode`… will fix separately